use_sim_time: false
a1_hardware_power_level: 6
stance_leg_control_type: 1 # 0: QP; 1: MPC

# physics parameters

a1_robot_mass: 12.5

#go1_lumped_inertia_xx: 0.0178533
#go1_lumped_inertia_xy: 0.0
#go1_lumped_inertia_xz: 0.0
#go1_lumped_inertia_yz: 0.0
#go1_lumped_inertia_yy: 0.0377999
#go1_lumped_inertia_zz: 0.0456542

go1_lumped_inertia_xx: 0.1486
go1_lumped_inertia_xy: 0.0005
go1_lumped_inertia_xz: -0.01
go1_lumped_inertia_yz: 0.0001
go1_lumped_inertia_yy: 0.3433
go1_lumped_inertia_zz: 0.0.3376

# default foothold position

go1_default_foot_pos_FR_x: 0.18
go1_default_foot_pos_FR_y: -0.125
go1_default_foot_pos_FR_z: -0.2

go1_default_foot_pos_FL_x: 0.18
go1_default_foot_pos_FL_y: 0.125
go1_default_foot_pos_FL_z: -0.2

go1_default_foot_pos_RR_x: -0.18
go1_default_foot_pos_RR_y: -0.125
go1_default_foot_pos_RR_z: -0.2

go1_default_foot_pos_RL_x: -0.18
go1_default_foot_pos_RL_y: 0.125
go1_default_foot_pos_RL_z: -0.2

a1_gait_counter_speed_FL: 1.0
a1_gait_counter_speed_FR: 1.0
a1_gait_counter_speed_RL: 1.0
a1_gait_counter_speed_RR: 1.0

# MPC parameters

q_weights_0: 150.0 # roll
q_weights_1: 150.0 # pitch
q_weights_2: 50.0 # yaw

q_weights_3: 0.0 # pos x
q_weights_4: 0.0 # pos y
q_weights_5: 80.0 # pos z

q_weights_6: 0.2 # omega x
q_weights_7: 0.2 # omega y
q_weights_8: 0.2 # omega z

q_weights_9:  0.3 # vel x
q_weights_10: 0.3 # vel y
q_weights_11: 0.3 # vel z

q_weights_12: 0.0

r_weights_0:  1e-2
r_weights_1:  1e-2
r_weights_2:  1e-3

r_weights_3:  1e-2
r_weights_4:  1e-2
r_weights_5:  1e-3

r_weights_6:  1e-2
r_weights_7:  1e-2
r_weights_8:  1e-3

r_weights_9:  1e-2
r_weights_10: 1e-2
r_weights_11: 1e-3

# foot swing controller parameters
# force_tgt = km_foot * kp_foot * foot_pos_error

a1_kp_foot_x: 120.0
a1_kp_foot_y: 120.0
a1_kp_foot_z: 80.0

a1_kd_foot_x: 6.0
a1_kd_foot_y: 6.0
a1_kd_foot_z: 5.0

a1_km_foot_x: 0.1
a1_km_foot_y: 0.1
a1_km_foot_z: 0.1
